airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -476,7 +475,13 @@ def __init__(
         number_logs_exception: int = 10,
         wait_for_completion: bool = True,
         waiter_delay: int = 6,
-        waiter_max_attempts: int = 100,
+        waiter_max_attempts: int = 1000000
+        * 365
+        * 24
+        * 60
+        * 10, 
```

Comment:
```suggestion
        waiter_max_attempts: int = 1000000 * 365 * 24 * 60 * 10,
```

To fix static checks